package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pick-your-movie",
+  "version": "1.0.0",
+  "description": "A web application to search for movies, with an autocomplete feature.",
+  "main": "src/sever.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node src/server.js"
+  },
```

Comment:
you should also have a script to run dev server with nodemon `"dev" : "nodemon src/server/js`"

Reply:
Yeah, this is very useful, thanks.